SimCalorimetry/HcalSimAlgos/src/HcalSiPM.cc

```diff
@@ -164,7 +174,15 @@ void HcalSiPM::setTemperatureDependence(double dTemp) {
 
 double HcalSiPM::cellCharge(double deltaTime) const {
   if (deltaTime <= 0.) return 0.;
-  if (deltaTime > 10./theTauInv) return 1.;
-  double result(1. - std::exp(-deltaTime*theTauInv));
+  if (deltaTime > 10.*theTau) return 1.;
```

Comment:
I was just trying to avoid multiple inversions of the tau parameter. I will run igprof to check if there is a performance penalty.